csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3352,6 +3352,184 @@ This does not exclude that such a document is also available in an access protec
 
 > Reasoning: If an advisory is already in the media, an end user should not be forced to collect the pieces of information from a press release but be able to retrieve the CSAF document.
 
+### 5.1.5 Requirement 5: security.txt
+
+In the security.txt there MUST be at least one field `CSAF` which points to either the ROLIE service document or a directory with CSAF files. If this field indicates a web URI, then it MUST begin with "https://" (as per section 2.7.2 of [RFC7230]). See [SECURITY-TXT] for more details.
+
+> At the time of this writing, the security.txt is still a proposed standard. The `CSAF` field has not been officially added yet.
+
+Examples:
+
+```
+CSAF: https://domain.tld/security/data/csaf/
+CSAF: https://psirt.domain.tld/advisories/csaf/
+CSAF: https://domain.tld/security/csaf/csaf-service.json
+```
+
+### 5.1.6 Requirement 6: DNS path
+
+The DNS record `csaf.data.security.domain.tld` SHALL resolve as a webserver which either serves directly the ROLIE service document or a directory with CSAF files.
+
+### 5.1.7 Requirement 7: One folder per year
+
+The CSAF documents must be located within folders named `<YYYY>` where `<YYYY>` is the year given in the value of `/document/tracking/initial_release_date`.
+
+Examples:
+
+```
+2021
+2020
+```
+
+### 5.1.8 Requirement 8: index.txt
+
+The index.txt file within MUST provide a list of all filenames of CSAF documents which are located in the sub-directories with their filenames.
+
+Examples:
+
+```
+2020/example_company_-_2020-yh4711.json
+2019/example_company_-_2019-yh3234.json
+2018/example_company_-_2018-yh2312.json
+```
+
+> This can be used to download all CSAF documents.
+
+### 5.1.9 Requirement 9: changes.csv
+
+The file changes.csv must contain the filename as well as the value of `/document/tracking/current_release_date` for each CSAF document in the sub-directories without a heading; lines must be sorted by the `current_release_date` timestamp with the latest one first.
+
+Examples:
+
+```
+2020/example_company_-_2020-yh4711.json, "2020-07-01T10:09:07Z"
+2018/example_company_-_2018-yh2312.json, "2020-07-01T10:09:01Z"
+2019/example_company_-_2019-yh3234.json, "2019-04-17T15:08:41Z"
+2018/example_company_-_2018-yh2312.json, "2019-03-01T06:01:00Z"
+```
+
+### 5.1.10 Requirement 10: Directory listings
+
+Directory listing SHALL be enabled to support manual navigation.
+
+### 5.1.11 Requirement 11: ROLIE service document
+
+Resource-Oriented Lightweight Information Exchange (ROLIE) is a standard to ease discovery of security content. ROLIE is built on top of the Atom Publishing Format and Protocol, with specific requirements that support publishing security content. The ROLIE service document MUST be a JSON file that conforms with [RFC8322] and lists the ROLIE feed documents.
```

Comment:
Note that Service Documents aren't specific to ROLIE, they are merely mentioned in RFC8322 as a way to discover ROLIE feeds. So technically this section should be named "AtomPub service document" with a requirement that it points to specific ROLIE CSAF feeds.

That was my point about requiring a service document. If as a vendor you publish only CSAF documents of one TLP level, your service document will be a pointer to that one feed and that's it, making it unnecessary since you could just point to the feed in the first place in e.g. security.txt.